### ISSUE TITLE
CoastSeg  is not compatible with pyTMD 2.1.7

### ISSUE THREAD

**Issue by 2320sharon** [SatelliteShorelines/CoastSeg#279]
When running tide correction with the latest pyTMD version the error below occurs due to changes in the pyTMD api

```
Traceback (most recent call last):
  File "c:\development\doodleverse\coastseg\CoastSeg\4_tide_correction.py", line 61, in <module>
    coastsegmap.compute_tidal_corrections(roi_ids, beach_slope, reference_elevation)
  File "C:\development\doodleverse\coastseg\CoastSeg\src\coastseg\coastseg_map.py", line 543, in compute_tidal_corrections  
    raise Exception(f"""Tide Model Error:\n {e}""")
Exception: Tide Model Error:
 cannot access local variable 'ph' where it is not associated with a value

```

**Comment by 2320sharon** [comment]
This is due to the fact that in the most recent up to pyTMD is `pyTMD.io.model` `format` property is no longer called `FES` but `netcdf-FES`

**Comment by 2320sharon** [comment]
This has been fixed in version 1.2.16
